en/docs/reference/mediators/respond-mediator.md

```diff
@@ -1,70 +1,64 @@
 # Respond Mediator
 
-The **Respond Mediator** stops the processing on the current message and sends the message back to the client as a response.
+The **Respond Mediator** stops the processing of the current message and sends the message back to the client as a response.
 
 ## Syntax
 
-The respond token refers to a \< `         respond        ` \> element,
-which is used to stop further processing of a message and send the
-message back to the client.
+The respond token refers to a `<respond/>` element, which is used to stop further processing of a message and send the message back to the client.
 
-``` java
+```xml
 <respond/>
 ```
 
 ## Configuration
 
-As with other mediators, after adding the Respond mediator to a
-sequence, you can click its up and down arrows to move its location in
-the sequence.
+As with other mediators, after adding the Respond mediator to a sequence, you can click its up and down arrows to move its location in the sequence.
```

Comment:
Let's remove this part. This seems to be related to EI management console. No longer relevant.